modules/tensor_mechanics/test/tests/lagrangian/total/action/action_1D.i

```diff
@@ -0,0 +1,100 @@
+# Simple 1D plane strain test
+
+[GlobalParams]
+  displacements = 'disp_x'
+  large_kinematics = true
+[]
+
+[Variables]
+      [./disp_x]
+      [../]
+[]
+
+[Mesh]
+  [./msh]
+    type = GeneratedMeshGenerator
+    dim = 1
+    nx = 10
+  []
+[]
+
+[Modules]
+  [./TensorMechanics]
+    [./Master]
+      [./all]
+        strain = FINITE
```

Comment:
I'm not sure how to clear it up, but the `ComputeLagranignanElasticSmallStress` uses the `ObjectiveStress` "option" and implements a small/engineering stress linear elastic stress update.  If you ask for small deformation kinematics that will just be the normal linear elasticity.  If you ask for large kinematics it will be hypoelasticity using your choice of objective rate.

The class names down this "route" for constitutive models should reflect the fact that they are implementing a small/engineering constitutive model.  i.e. that's what the person implementing the model provides -- the small strain/stress constitutive model -- and the system will magically advect things using an objective integration for large deformations.

Would naming these types of models "Engineering" make more sense?  i.e. `ComputeLagrangianElasticEngineeringStress`?  Or I'm open to other suggestions.

Reply:
Thanks for the details. Yeah, I think using "Engineering" would help here. `Compute(Lagrangian)LinearElasticObjectiveStress` would be another option?

Reply:
That would make the most sense from a straight naming point of view, but I worry it will make people think they need to return the integrated objective stress instead of the small/engineering stress.  I'll wait for opinions but I'm okay with either naming scheme.

Reply:
I went with `Engineering` at least for now.